op-mode-definitions/include/isis-common.xml.i

```diff
@@ -54,6 +54,30 @@
   </properties>
   <command>${vyos_op_scripts_dir}/vtysh_wrapper.sh $@</command>
 </tagNode>
+<node name="mpls">
+  <properties>
+    <help>Show IS-IS MPLS specific information</help>
+  </properties>
+  <children>
+    <node name="ldp-sync">
+      <properties>
+        <help>Show IS-IS LDP-IGP synchronization information</help>
+      </properties>
+      <command>${vyos_op_scripts_dir}/vtysh_wrapper.sh $@</command>
+      <children>
+        <tagNode name="interface">
+          <properties>
+            <help>Show specific IS-IS LDP-IGP synchronization for an interface</help>
```

Comment:
Can we probably remove IS-IS and OSPF from the help string so we can make this an includable XML snipped to become more and more single-source?

Reply:
Sure of course. Will make those changes.